### DISCUSSION TITLE
Block#726859 of calibration testnet is empty

### DISCUSSION THREAD

**Discussion by rtj599** [filecoin-project/community#653] [Developers]
this block is empty on calibration testnet:
https://calibration.filscout.com/en/tipset/height/726859

Why it happened and could it happen on mainnet?

**Comment by wolfduan** [comment]
github

**Comment by rtj599** [comment]
https://github.com/filecoin-project/community/issues/654

**Comment by rjan90** [comment]
Hey @rtj599! Yes, the mainnet can also have "null rounds" - i.e periods of time where there are no block produced. On the mainnet these occur very infrequently, and usually can only be seen during network upgrades with heavier migrations.

**Reply by rtj599** [reply]
got it, thanks!
